tests/testthat/test-methods-epi_df.R

```diff
@@ -0,0 +1,21 @@
+test_that("head and tail do not drop the epi_df class", {
+  tib <- tibble::tibble(
+    x = 1:10,
+    y = 1:10,
+    time_value = rep(seq(
+      as.Date("2020-01-01"),
+      by = 1,
+      length.out = 5
+    ), times = 2),
+    geo_value = rep(c("ca", "hi"), each = 5)
+  ) %>% epiprocess::as_epi_df()
+  
+  expect_true(is_epi_df(head(tib)))
+  expect_true(is_epi_df(tail(tib)))
+  expect_identical(attributes(head(tib))$metadata$geo_type, attributes(tib)$metadata$geo_type)
+  expect_identical(attributes(head(tib))$metadata$time_type, attributes(tib)$metadata$time_type)
+  expect_identical(attributes(head(tib))$metadata$as_of, attributes(tib)$metadata$as_of)
```

Comment:
Is `additional_metadata` also supposed to be preserved?  The tests for head and tail here should probably ensure that the desired behavior (retaining, dropping, etc.) is achieved.

Reply:
Sure I could add a column for that... I haven't seen it done elsewhere (ex. in the other tests). So, let me know what a reasonable example of `additional_metadata` would be and I can put that in.

Reply:
See `other_keys` / `source_url` example below.  Marking this resolved so just that comment remains to look at.